PyInstaller/depend/imphook.py

```diff
@@ -16,7 +16,7 @@
 
 import glob, sys, weakref
 import os.path
-
```

Comment:
Keep a blank line here - otherwise the linter gets sad...

Reply:
Also, remember our no-functional-changes policy.